website/pages/style.css

```diff
@@ -15,22 +15,20 @@
 @import '~@hashicorp/react-button/dist/style.css';
 @import '~@hashicorp/react-call-to-action/dist/style.css';
 @import '~@hashicorp/react-case-study-slider/dist/style.css';
-@import '~@hashicorp/react-code-block/dist/style.css';
+@import '~@hashicorp/react-code-block/style.css';
 @import '~@hashicorp/react-consent-manager/dist/style.css';
 @import '~@hashicorp/react-content/dist/style.css';
 @import '~@hashicorp/react-docs-page/style.css';
 @import '~@hashicorp/react-docs-sidenav/dist/style.css';
-@import '~@hashicorp/react-enterprise-alert/dist/style.css';
+@import '~@hashicorp/react-enterprise-alert/style.css';
```

Comment:
Despite being unrelated components, I had to update these paths to remove the `/dist` to avoid breakage. See [`nextjs-scripts@13.2.0` release notes](https://github.com/hashicorp/nextjs-scripts/releases/tag/13.2.0) for details.